packages/broker/test/unit/ConfigWizard.test.ts

```diff
@@ -174,16 +170,28 @@ describe('ConfigWizard', () => {
         it ('should exercise the plugin port assignation path with a stringified number', () => {
             assertValidPort('3737')
         })
+    })
+
+    describe('identity', () => {
+        it ('happy path', () => {
+            const privateKey = '0x9a2f3b058b9b457f9f954e62ea9fd2cefe2978736ffb3ef2c1782ccfad9c411d'
+            const identity = getNodeIdentity(privateKey)
+            expect(identity.mnemonic).toBe('Mountain Until Gun')
+            expect(identity.networkExplorerUrl).toBe('https://streamr.network/network-explorer/nodes/0x909DC59FF7A3b23126bc6F86ad44dD808fd424Dc')
+        })
+    })
 
+    describe('end-to-end', () => {
         it ('should exercise the happy path with default answers', () => {
-            const answers = {
-                generateOrImportEthereumPrivateKey: 'Generate',
+            const privateKeyAnswers = {}
+            const pluginsAnswers = {
                 selectPlugins: [ 'websocket', 'mqtt', 'publishHttp' ],
-                websocketPort: DEFAULT_CONFIG_PORTS.DEFAULT_WS_PORT,
```

Comment:
The variable name is bit misleading as the ports are not defaults only for the config, but for the Broker or the IT industry (the MQTT port)

Reply:
The naming here is due to where the imports come from:

```typescript
const DEFAULT_WS_PORT = WebsocketConfigSchema.properties.port.default
const DEFAULT_MQTT_PORT = MqttConfigSchema.properties.port.default
const DEFAULT_HTTP_PORT = BrokerConfigSchema.properties.httpServer.properties.port.default
const DEFAULT_LEGACY_WS_PORT = LegacyWebsocketConfigSchema.properties.port.default
``

Reply:
Broker stores the default values in the schema file so that users can easily override the values. If there wouldn't be any configuration options, we'd definitely run the MQTT plugin at port 1883.